resources/src/main/java/org/robolectric/res/android/FileMap.java

```diff
@@ -31,7 +31,6 @@ public class FileMap {
 
   private ZipFile zipFile;
   private ZipEntry zipEntry;
-  private boolean readOnly;
```

Comment:
It's not recommended to remove `readOnly`, because it has been initialized. I don't know whether we will leverage it.

Reply:
ahh, thats fine we can suppress it as well

Reply:
https://github.com/robolectric/robolectric/pull/7374/files#diff-602eec5b436befc587db2bb2898c53ab5e686c8670d514120ddf70b5b1989544R35